help/destinations/catalog/email-marketing/mailchimp-interest-group.md

```diff
@@ -0,0 +1,213 @@
+
+[!DNL Mailchimp Interest Group] uses OAuth 2 with Authorization Code to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Mailchimp Interest Group] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Mailchimp Interest Groups] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The sales department of a sporting good website wants to broadcast an email based marketing campaign to a list of contacts who have self identified themselves as being interested in soccer. The lists of contacts are segregated as batches in the data export received from the development team of the website and therefore need to be tracked. The team identifies an existing [!DNL Mailchimp] audience and starts building the Experience Platform segment's into which the contacts from each list are added. After sending these segments to [!DNL Mailchimp Interest Groups] if any contacts do not exist in the selected [!DNL Mailchimp] audience they get added to a group having the segment name the contact belongs to. If any contacts already exist in the [!DNL Mailchimp] audience or group then their information is updated. Once the data is sent over to [!DNL Mailchimp Interest Group] the Sales team can select and send the marketing campaign email to the soccer interest group within the [!DNL Mailchimp] audience.
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that you need to set up in Experience Platform and [!DNL Mailchimp] and for information that you need to gather before working with the [!DNL Mailchimp Interest Group] destination.
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Mailchimp Interest Group] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+### Prerequisites for the [!DNL Mailchimp Interest Group] destination {#prerequisites-destination}
+
+Note the following prerequisites in order to export data from Platform to your [!DNL Mailchimp Interest Group] account:
+
+#### You need to have a [!DNL Mailchimp] account {#prerequisites-account}
+
+Before you can create a [!DNL Mailchimp Interest Group] destination, you must first ensure that you have a [!DNL Mailchimp] account. If you do not have one already visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.
```

Comment:
comma:

If you do not have one already, visit the [[!DNL Mailchimp] signup page](https://login.mailchimp.com/signup/) to register and create your account.